include/config.js

```diff
@@ -219,6 +219,7 @@ Configuration.getBaseConfig = function(multisite) {
         settings: {
             use_memory: true,
             use_cache: false,
+            autoSync: false,
```

Comment:
If you could, please include a line or two about how it works etc.  This just helps us keep the source documented and ensures that it evolves with the code.